applications/MultilevelMonteCarloApplication/tests/test_xmcAlgorithm.py

```diff
@@ -184,8 +194,13 @@ def test_mlmc_Kratos(self):
             "poisson_square_2d/problem_settings/parameters_xmc_test_mlmc_Kratos_asynchronous_poisson_2d_with_combined_power_sums_multi_ensemble.json",
             "poisson_square_2d/problem_settings/parameters_xmc_test_mlmc_Kratos_asynchronous_poisson_2d_DAR.json",
             "poisson_square_2d/problem_settings/parameters_xmc_test_mlmc_Kratos_asynchronous_poisson_2d_fixedsamples.json",
+            # "poisson_square_2d/problem_settings/parameters_xmc_test_mlmc_Kratos_asynchronous_adaptivefixednumberlevels_poisson_2d.json",
+            # "poisson_square_2d/problem_settings/parameters_xmc_test_mlmc_Kratos_adaptivefixednumberlevels_poisson_2d.json"
```

Comment:
remove?

Reply:
Ah actually these are tests we should run

Reply:
I forgot they were commented :/

Reply:
Let me check again nightly